TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoLinearAPI.kt

```diff
@@ -0,0 +1,91 @@
+package org.firstinspires.ftc.teamcode.api.linear
+
+import com.qualcomm.robotcore.util.Range
+import org.firstinspires.ftc.teamcode.api.TriWheels
+import org.firstinspires.ftc.teamcode.core.API
+import kotlin.math.atan2
+import kotlin.math.sqrt
+
+/**
+ * This is a demo API for driving related to april tags.
+ *
+ * Requires the [TriWheels] API.
```

Comment:
APIs now have a dependency system so it will throw an exception if you initialize this and not `TriWheels`. Could you please override `this.dependencies` with a [set](https://kotlinlang.org/docs/collections-overview.html#set) containing `TriWheels`?

Reply:
I can not figure out how to write this line correctly.
I have tried different variations of:
override fun this.dependencies() = setOf(TriWheels)
override this.dependencies() = setOf(TriWheels)

Reply:
This should work:

```kotlin
override val dependencies = setOf(TriWheels)
```